src/controllers/midi/midicontroller.h

```diff
@@ -66,7 +67,7 @@ class MidiController : public Controller {
         send(data);
     }
 
-    void makeInputHandler(const QJSValue& control, const QJSValue& scriptCode);
+    void makeInputHandler(const QString& group, int status, int midino, const QJSValue& scriptCode);
```

Comment:
I still think that the group is unnecessary for mappings declared from JS.

```suggestion
    void makeInputHandler(int status, int midino, const QJSValue& scriptCode);
```